tower-http/CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
-None.
+- Clarify exactly when each `Trace` callback is called.
+- Call `Trace`'s `on_failure` callback each time the response body produces an
+  error (in `http_body::Body::poll_data`). It would previously only be called
+  the first time.
```

Comment:
Note to self: Remove this line. It's wrong. It is only called the first time poll_data fails.